neusomatic/python/extract_postprocess_targets.py

```diff
@@ -107,16 +107,53 @@ def extract_postprocess_targets(reference, input_vcf, min_len, max_dist, extend_
             if not record_set:
                 record_set.append(record)
                 continue
-            chrom_, pos_, ref_, alt_ = push_left_var(
-                ref_fasta, chrom, pos, ref, alt)
-            if len(list(filter(lambda x: (chrom == x[0] and
-                                          (min(abs(x[1] + len(x[2]) - (pos + len(ref))),
-                                               abs(x[1] - pos),
-                                               abs(min(x[1] + len(x[2]), pos + len(ref)) - max(x[1], pos))) <= max_dist)), record_set))) > 0 or len(
-                list(filter(lambda x: (chrom_ == x[0] and
-                                       (min(abs(x[1] + len(x[2]) - (pos_ + len(ref_))),
-                                            abs(x[1] - pos_),
-                                            abs(min(x[1] + len(x[2]), pos_ + len(ref_)) - max(x[1], pos_))) <= max_dist)), record_set))) > 0:
+            chrom_, pos_, ref_, alt_ = push_left_var(ref_fasta, chrom, pos, ref, alt)
+            if (
```

Comment:
@pavanstrand the formatting seems to have gone crazy here. Would be better with a bit of refactoring here.